templates/plugin/mcelog.conf.erb

```diff
@@ -0,0 +1,10 @@
+<Plugin mcelog>
```

Comment:
can you please convert this to an epp template? We require this for new templates. For our other review guidelines please have a look at https://voxpupuli.org/docs/reviewing_pr/

Reply:
Converted to epp template